packages/fxa-settings/src/components/Settings/DropDownAvatarMenu/index.tsx

```diff
@@ -31,6 +32,7 @@ export const DropDownAvatarMenu = () => {
   const signOut = async () => {
     if (session.destroy) {
       try {
+        firefox.fxaLogout({ uid });
```

Comment:
This is a little confusing and we used to have tons of linked issues around it. But basically, does signing out of Settings mean you sign out of Firefox?

In the past, if the user was coming from Sync relier, then this would sign them out. I think we can hold off on adding this in here until we confirm with product what we want it to do.

Reply:
Removed the fxaLogout call.